src/frontend/src/btc/components/transactions/BtcTransactionModal.svelte

```diff
@@ -58,9 +59,21 @@
 	sendToLabel={$i18n.transaction.text.to}
 	typeLabel={type === 'send' ? $i18n.send.text.send : $i18n.receive.text.receive}
 >
-	<!--	TODO: Implement BtcTransactionStatus component	-->
+	{#if nonNullish(confirmations)}
+		<Value ref="confirmations" slot="confirmations">
```

Comment:
This compiles? Shouldn't the `slot=confirmations` be provided with a `svelte:fragement` and the `if nonNullish(confirmation` find place within the fragment?

Reply:
Yes, it works fine, it's the way how it's used in other components too. I think it would be required to do like you said if there wasn't a wrapper component (`Value` in this case).

Reply:
TIL I did not knew that. Really thought the linter would complain about that. Coolio, thx for letting me know.

Reply:
Sorry, my bad - I checked it once again and it does have to be done like you said. Wondering how it worked for me the first time. Even though Svelte is a really nice instrument, I still find some of its concepts a bit confusing 🙃

Reply:
Wait for it, in Svelte v5 they removed slots 🤪